.gitignore

```diff
@@ -1,7 +1,36 @@
+# Ignore certain files #
+
+# CMake - General Files #
+
+cmake/
```

Comment:
There's already a directory in the repository called ``cmake/``, what is this ignoring?